packages/testsuite/cypress/support/form-editing.ts

```diff
@@ -163,10 +163,19 @@ Cypress.Commands.add("clearListAttributeItems", (attributeName) => {
     });
 });
 
-Cypress.Commands.add("selectText", (formId, attributeName, value) => {
-  cy.formInput(formId, attributeName).select(value, { force: true });
-  cy.formInput(formId, attributeName).should("have.value", value);
-  cy.formInput(formId, attributeName).trigger("change", { force: true });
+Cypress.Commands.add("selectInDropdownMenuOnPage", (elementId, value) => {
+  // somtimes can be the dropdown menu behind a pop-up alert notification. So we need close them.
+  cy.closeAllPopUpNotifications();
+  cy.get(`button[data-id="${elementId}"]`).click();
+  cy.get(`button[data-id="${elementId}"]`)
+    .parent()
+    .within(() => {
+      cy.get(`a.dropdown-item`).contains(value).click();
+    });
+});
+
+Cypress.Commands.add("selectInDropdownMenuInForm", (formId, fieldName, value) => {
```

Comment:
> selectInDropdownMenuInForm

Can we use just selectInDropdownMenu as it's the most common case and we are in form-editing utils?